test/run

```diff
@@ -49,7 +49,7 @@ function assert_svg {
 # checks the dimensions of an image, requires ImageMagick
```

Comment:
This is no longer true then, correct? (Yay!)

🤔 Does Ubuntu come with ImageMagick by default? Otherwise I don't understand how this ever worked in the first place.

Reply:
Good catch! I guess it is a dependency of one of the thousands of packages GitHub preinstalls on the runners:

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Reply:
Yikes! TIL - not sure whether to thank or curse you...